### DISCUSSION TITLE
Safari compatibility

### DISCUSSION THREAD

**Discussion by soda-pop-ice-cream** [bluenviron/mediamtx#1080] [Questions] [answered]
Hi there!
I wanted to ask a questinon about how can i make web stream compactible for both DASH and HLS clients? I mean, if client connecting with Safari browser(or Apple devices in general) then it should play low latency HLS stream, if any other browser/device then it would get DASH stream?
From reading documentation, it seems that such behavior could be archived by some actions with ffmpeg and .m3u8, but i just can't get how to achive this.

Currently, if i don't specify any parameters in config i have DASH stream avaliable at example.org/path, and if i enable HLS via:
```
hlsVariant: lowLatency
hlsEncryption: yes
hlsServerKey: server.key
hlsServerCert: server.crt
```
then it start work on Apple devices, but breaks on others.

**Comment by aler9** [comment, marked as answer]
Hello, HLS is compatible with all browsers and devices, since [hls.js](https://github.com/video-dev/hls.js/) is used as polyfill. We don't plan to support DASH at the moment.

[a comment by an automated account is not shown]
